### ISSUE TITLE
Hovering on tab should make the text $tc-black

### ISSUE THREAD

**Issue by vic-topcoder** [appirio-tech/connect-app#2153]
### Expected behavior
When I hove on the primary tabs, the text of the tab should transition to $tc-black (the same as active tab) without the underline. Only active tab would have the underline

### Actual behavior
Hover state is the same as default state

**Comment by maxceem** [comment]
@vic-topcoder I think we should make the same behaviour for Phase tabs as it looks same and they use the same component:

![image](https://user-images.githubusercontent.com/146016/46389086-9d6f9e00-c702-11e8-9e6e-c9b07e9ee908.png)

Let me know if it's not desired in some reason.

**Comment by vic-topcoder** [comment]
Yes, all tabs should behave the same (consistency). Great observation @maxceem

**Comment by sumitdaga** [comment]
@maxceem 
please assign this to me!

**Comment by sumitdaga** [comment]
btw there is no cf13 branch i think as of now!

**Comment by maxceem** [comment]
@sumitdaga thanks for pointing that out, added https://github.com/appirio-tech/connect-app/tree/cf13

**Comment by sumitdaga** [comment]
@maxceem 
ready for review
PR #2592
